testing/p0f/p0f-fixes.patch

```diff
@@ -0,0 +1,42 @@
+--- p0f-3.09b.orig/build.sh
```

Comment:
Could you add a comment at the start of the file explaining what it fixes and why it's needed? If it's too many different changes, it may be useful splitting up as well.